### PR TITLE
Add new transition api URl

### PR DESCRIPTION
**Pull request by szok**
- Add new url to transition API, example: /api/app_label/model/<pk>/transition/<transition_id>

### CODE REVIEW

**Comment by ghost** [comment]
:ok:

File: src/ralph/lib/transitions/api/views.py [hunk 1]
```diff
@@ -179,3 +190,34 @@ def post(self, request, *args, **kwargs):
...
+
+class TransitionView(TransitionViewMixin):
+
+    def initial(self, request, *args, **kwargs):
+        try:
+            filters = {
+                'model__content_type__model': kwargs['model'],
+                'model__content_type__app_label': kwargs['app_label'],
+            }
+            if kwargs.get('transition_pk'):
```
Comment (mkurek) [inline src/ralph/lib/transitions/api/views.py]: is it possible to have transition_pk here?

File: src/ralph/lib/transitions/api/views.py [hunk 2]
```diff
@@ -179,3 +190,34 @@ def post(self, request, *args, **kwargs):
...
+            raise NotFound('Transition not found!')
+        super().initial(request, *args, **kwargs)
+
+
+class TransitionByIdView(TransitionViewMixin):
+
+    def initial(self, request, *args, **kwargs):
+        try:
+            self.transition = Transition.objects.get(
+                pk=kwargs['transition_pk']
```
Comment (mkurek) [inline src/ralph/lib/transitions/api/views.py:232]: if this kwarg is required, add is to method declaration

File: src/ralph/lib/transitions/api/views.py [hunk 3]
```diff
@@ -179,3 +190,34 @@ def post(self, request, *args, **kwargs):
...
+class TransitionView(TransitionViewMixin):
+
+    def initial(self, request, *args, **kwargs):
+        try:
+            filters = {
+                'model__content_type__model': kwargs['model'],
+                'model__content_type__app_label': kwargs['app_label'],
+            }
+            if kwargs.get('transition_pk'):
+                filters['pk'] = kwargs['transition_pk']
```
Comment (mkurek) [inline src/ralph/lib/transitions/api/views.py]: if this kwarg is required, add is to method declaration

File: src/ralph/lib/transitions/api/views.py [hunk 4]
```diff
@@ -179,3 +190,34 @@ def post(self, request, *args, **kwargs):
             ]
             status_code = status.HTTP_202_ACCEPTED
         return Response(result, status=status_code)
+
+
+class TransitionView(TransitionViewMixin):
```
Comment (mkurek) [inline src/ralph/lib/transitions/api/views.py:201]: add some docstring about use-case of this view (class)

File: src/ralph/lib/transitions/api/views.py [hunk 5]
```diff
@@ -179,3 +190,34 @@ def post(self, request, *args, **kwargs):
...
+
+    def initial(self, request, *args, **kwargs):
+        try:
+            filters = {
+                'model__content_type__model': kwargs['model'],
```
Comment (mkurek) [inline src/ralph/lib/transitions/api/views.py:213]: if this kwarg is required, add is to method declaration

**Comment by mkurek** [comment]
Please add some entry to docs about using transitions through API.

File: src/ralph/lib/transitions/api/views.py (hunk 4 above)
Comment (szok) [inline src/ralph/lib/transitions/api/views.py:201]: Done

**Comment by mkurek** [comment]
:ok:

**Pull request merged**